src/ralph/assets/tests/test_api.py

```diff
@@ -464,6 +464,15 @@ def test_lte_polymorphic(self):
         response = self.client.get(url, format='json')
         self.assertEqual(len(response.data['results']), 2)
 
+    def test_is_lookup_used(self):
+        url = '{}?{}'.format(
+            reverse('baseobject-list'), urlencode(
+                {'hostname__icontains': 'testtest'}
```

Comment:
Could it be:
` {'hostname__icontains': 'testtest'}` -> `` {'hostname__icontains': 'unexisted-host'}`
or anything more descriptive, it would increase this test readabilty.